illink.targets

```diff
@@ -25,6 +25,12 @@
     <ILLinkRewritePDBs Condition="'$(ILLinkRewritePDBs)' == '' AND '$(DebugSymbols)' != 'false'">true</ILLinkRewritePDBs>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(ILLinkTrimXml)' != ''">
```

Comment:
FWIW the condition is not necessary, since if this property is blank, the expression below should create no items.